src/javascript/app/pages/trade/barriers.js

```diff
@@ -27,9 +26,10 @@ const Barriers = (() => {
             const unit     = getElementById('duration_units');
             const end_time = getElementById('expiry_date');
             const is_daily = (unit && isVisible(unit) && unit.value === 'd') ||
-                (end_time && isVisible(end_time) && moment(end_time.getAttribute('data-value')).isAfter(moment(), 'day'));
+                (end_time && isVisible(end_time));
```

Comment:
Showing barriers even if the end-time is the current date.

Reply:
this is the check for `is_daily` value, if we remove that check then `is_daily` is not being decided correctly. we use this to show intraday or daily barrier values and need to keep this check.

Reply:
I presume `is_daily` means contracts lasts 1 day or more, so with this change wouldn't `is_daily` loose it's semantic meaning? So for other contracts with end time `is_daily` would be true always, even if end time is in one minute or something.

Reply:
(oops, duplicate)